### PR TITLE
update support new device docs

### LINKED ISSUE #6962: need to update support_new_device doc

### PR DESCRIPTION
**Pull request by QiJune**
Fix #6962

### CODE REVIEW

File: doc/design/support_new_device.md [hunk 1]
```diff
@@ -25,13 +25,14 @@ There are mainly three parts that we have to consider while integrating a new de
 
 ### Place and DeviceContext
 
+Please remind that device and computing library are not one-to-one corresponding. A device can have a lot of computing libraries and a computing library can also support several devices.
 
 #### Place
-Fluid uses class [Place](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/platform/place.h#L55) to represent different devices and computing libraries. There are inheritance relationships between different kinds of `Place`.
+Fluid uses class [Place](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/platform/place.h#L55) to represent the device memory where data is locating. If we add another device, we have to add corresponding `DevicePlace`.
```
Comment (abhinavarora) [inline doc/design/support_new_device.md]: where data is locating -> where data is **located**

Comment (QiJune) [reply doc/design/support_new_device.md]: Done

File: doc/design/support_new_device.md [hunk 2]
```diff
@@ -106,7 +107,7 @@ template <typename Place>
 size_t Used(Place place);
 ```
 
-To implementing these interfaces, we have to implement MemoryAllocator for different Devices
+To implementing these interfaces, we have to implement MemoryAllocator for different Devices.
```
Comment (abhinavarora) [inline doc/design/support_new_device.md]: implementing -> **implement**

Comment (QiJune) [reply doc/design/support_new_device.md]: Done

**Pull request merged**